build-ps/rpm/mysql-5.6-sharedlib-rename.patch

```diff
@@ -8,7 +8,7 @@ diff -rup old/client/CMakeLists.txt new/client/CMakeLists.txt
 -TARGET_LINK_LIBRARIES(mysql perconaserverclient)
 +TARGET_LINK_LIBRARIES(mysql mysqlclient)
```

Comment:
That's not a change, it's part of the centos7 patch so github gui is marking it red/green since it contains +/- start of line.